server/app/routes/api/v1/index.js

```diff
@@ -3,10 +3,15 @@ var router = require('express').Router();
 module.exports = router;
 
 router.use('/users', require('./users'));
+router.use('/teams', require('./teams'));
+router.use('/organizations', require('./organizations'));
 router.use('/annotations', require('./annotations'));
-router.use('/questions', require('./assessments/questions'));
+router.use('/questions', require('./questions/questions'));
```

Comment:
Questions seem to be the pieces that make up an assessment, whereas to me responses are what make up an evaluation? 

It seems that you have it so that each question is linked to a user and an assessment so each question has an individual response? If it is that each question has multiple responses that are linked to the question and a user, then to me questions make up assessments and might be better fit inside of that folder. This might be too philosophical though